include/Array.h

```diff
@@ -387,7 +387,7 @@ namespace cpp
 
 // --- Array_obj ------------------------------------------------------------------
 //
-// The Array_obj specialises the ArrayBase, adding typing where required
+// The Array_obj specializes the ArrayBase, adding typing where required

```

Comment:
This one was fine, we can BE instead of AE.

Reply:
Alright ill revert it while i fix more typos